cmd2.py

```diff
@@ -113,6 +112,12 @@ def __subclasshook__(cls, C):
 else:
     from contextlib import redirect_stdout, redirect_stderr
 
+if sys.version_info > (3, 0):
```

Comment:
FYI, using the ``six`` module can help avoid Python 2 vs 3 compatibility gymnastics such as this.

``six`` is a Python 2 vs 3 compatibility module (6 == 2 * 3 == 3 * 2).